script.js

```diff
@@ -1,4 +1,275 @@
+//Create a version of the Beat That dice game, where players create the largest number they can by concatenating random dice roll digits.
+
+/* 
+Rules:
+
+1) There are 2 players and players take turns.
+
+2) When a player clicks Submit, the game rolls 2 dice and shows the dice rolls, for example 3 and 6.
+
+3) The player picks the order of the dice they want. For example, if they wanted the number 63, they would specify that the 2nd dice goes first. You can choose how the player specifies dice order.
+
+4) After both players have rolled and chosen dice order, the player with the higher combined number wins.
+*/
+
+//Create global variables that stores whether the players have rolled
+var hasPlayer1Rolled = 0;
+var hasPlayer2Rolled = 0;
+
+//Create a global variable that is an array that will store the players' rolled numbers. It is empty at first but when the numbers are pushed in it will be in the following format ideally: [player1Dice1, player1Dice2] [player2Dice1, player2Dice2]
+var player1DiceRolls = [];
+var player2DiceRolls = [];
+
+//Create global variables for the players' concatenated numbers
+var player1ConcatenatedNumber = 0;
+var player2ConcatenatedNumber = 0;
+
+//Create a global variable to store the reset mode
+var timeToReset = 0;
```

Comment:
Nice and clear global variables. For `hasPlayer1Rolled` and `hasPlayer2Rolled`, since they can only take values 0 and 1, you are implicitly using a boolean variable (more formally, 0 is falsy and 1 is truthy, see [here](https://developer.mozilla.org/en-US/docs/Glossary/Truthy)